quark/core/axmlreader/__init__.py

```diff
@@ -104,6 +108,7 @@ def __init__(self, file_path, core_library="rizin", structure_path=None):
             self._core = r2pipe.open(file_path)
 
         self._core.cmd(f"pfo {structure_path}")
+        file_manager.close()
```

Comment:
Could you use `ExitStack` with a `with` statement to make the code clear?

For example,
```python
with ExitStack() as stack:
    # The code goes here
```

[Reference](https://docs.python.org/3/library/contextlib.html#contextlib.ExitStack)